src/view/cli.js

```diff
@@ -19,16 +20,33 @@ const styles = StyleSheet.create({
   },
 });
 
-const CLIView = ({ store, nav }) => (
```

Comment:
We should inject the `log` instance of the LogActions as a prop into this view via `({ store, nav, log })`. And inject `log` in https://github.com/lightninglabs/lightning-app/blob/1ee33ed1651bd84a87174a5ebc3c16b27c0e030f/src/view/main-mobile.js#L140

Reply:
Right now all of the log action's functions are standalone and individually exported & imported. Should they all be moved into the Log class and called like that?